raw_ioTypes.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef raw_io_TYPES_HPP
+#define raw_io_TYPES_HPP
+
+#include <base/Float.hpp>
+#include <stdint.h>
+
+namespace raw_io {
+
+    struct TemperatureSensorSettings {
+        float min_voltage = base::unknown<float>();
+        float max_voltage = base::unknown<float>();
+        float min_temperature = base::unknown<float>();
+        float max_temperature = base::unknown<float>();
```

Comment:
Are these really the min and max measurements? 
If they really are, shouldn't we discard analog reading outside this range?